src/extension.ts

```diff
@@ -565,23 +583,25 @@ export function registerClientViews(context: ExtensionContext): void {
     PROGRESS_NOTIFICATION_ID,
     (objectResult: PlainObject<LanguageClientProgressResult>) => {
       try {
-        if (
-          isPlainObject(objectResult.value) &&
-          objectResult.value.kind !== 'end'
-        ) {
-          resetExecutionTab(resultsTreeDataProvider, resultsViewprovider);
+        if (resultsViewprovider.getView() && resultsViewprovider.getWebView()) {
```

Comment:
Had to add this check here because this callback is also called when opening query builder (not just results view), and there's no `resultsViewprovider` in that case, so we were showing an unnecessary error message.